src/Cli/dotnet/OptionForwardingExtensions.cs

```diff
@@ -59,10 +59,6 @@ public ForwardedOption(Option option) : base(option.Aliases.ToArray(), option.De
                 {
                     Argument = option.Argument;
                 }
-                if (option.GetSuggestions() != null)
```

Comment:
Just delete without replacement?

Reply:
Yes, that's why I moved a few suggestions from options to arguments ( see https://github.com/dotnet/sdk/pull/15499/files#diff-15c09ffd510898e37c1ae177803c544ee05ad9bfa2a1b8a47b60284539e3bcd3L130 for example). It's functionally the same except now since none of the options have suggestions we don't have to make this GetSuggestions call, which is really slow since it calls all the delegates.